lib/apphook.c

```diff
@@ -98,21 +98,18 @@ static void
 _update_current_state(gint type)
 {
   if (AH_REOPEN == type)
-     return;
```

Comment:
ahh.. so you did not run the style-format for the first patch :)

Reply:
:(